maven-surefire-plugin/src/site/apt/examples/junit-platform.apt.vm

```diff
@@ -411,6 +411,53 @@ Using JUnit 5 Platform
 </dependencies>
 +---+
 
+** JUnit5 Suite
+
+  For more information see this
+  {{{https://github.com/apache/maven-surefire/tree/master/surefire-its/src/test/resources/junit5-suite}example}}.
+
++---+
+<dependencies>
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>5.8.0</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.junit.platform</groupId>
+        <artifactId>junit-platform-suite-api</artifactId>
+        <version>1.8.0</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+                <includes>
+                    <include>JUnit5Tests</include>
+                </includes>
+            </configuration>
+            <dependencies>
+                <dependency>
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter-engine</artifactId>
+                    <version>5.8.2</version>
+                </dependency>
+                <dependency>
+                    <groupId>org.junit.platform</groupId>
+                    <artifactId>junit-platform-suite-engine</artifactId>
+                    <version>1.8.2</version>
+                </dependency>
+            </dependencies>
+        </plugin>
+    </plugins>
+</build>
```

Comment:
Looks like next complicated ... why dependency in project and plugin
Even if we we support it  in code for some reason we shouldn't give such examples ... project dependency should be enough.

Maybe we should also link to JUnit documentation.
https://junit.org/junit5/docs/current/user-guide/#junit-platform-suite-engine

Reply:
Because there are more alternatives. So the JUnit5 team separated API from impl and they always stated that the user doe not need to access the impl of the engine.
If you like I put engines to the dependencies. Both would work. It's just matter of taste.

Reply:
IMHO the users don;t care if they have an access to the engine internals. So I wil use only project deps. thx

Reply:
agree with @slawekjaranowski such extra configuration complexity will be confusing for users. and if given as an example in the users documentation the users will simply copy/paste this over complicated configuration whereas they do not need it.